SDWebImage/SDImageCache.h

```diff
@@ -160,6 +160,17 @@ typedef void(^SDWebImageCalculateSizeBlock)(NSUInteger fileCount, NSUInteger tot
  */
 - (void)storeImageDataToDisk:(nullable NSData *)imageData forKey:(nullable NSString *)key;
 
+#pragma mark - Update Ops
+
+/**
+ * Update image memory cost if image already be stored in memory.
+ *
+ *
+ * @param key        The unique image cache key, usually it's image absolute URL
+ * @note After image be stored in memory cache, if image's memory size changes, such as set a large memory associated object to it, or subclass `UIImage` which has its own large memory properties, you can use this to update memory cost which can make memory cache handle `maxMemoryCost` correctly
+ */
+- (void)updateImageMemoryCostWithKey:(nullable NSString *)key;
```

Comment:
This introduced API is useless. Is this really helpful for current user (Or for your own framework business code ?)
If the answer is NO, delete it and use other way instead (private header, or inline method)

Reply:
Don't be too subjective. If we provide setter method which users can update cost, this method is needed, because update cost needs at least three steps, verify wether image in cache, if it exist, remove it and reinsert it. We need to encapsulate these things.